index.html

```diff
@@ -2532,183 +2532,169 @@ <h1>Servient Implementation</h1>
             shown in <a href="#architecture-implementation"></a>.
-            This figure shows and implementation that is using the optional 
-            <a>WoT Scripting API</a>. 
-            Scripts are actually executed by a nested Scripting Runtime, 
-            while WoT-specific Servient Implementation aspects are managed
-            by the WoT Runtime. 
-            <a>Consumed Thing</a> and <a>Exposed Thing</a>
-            objects provide abstract interfaces to the Things
-            described in related Thing Descriptions. Scripts can
-            operate on Things using these abstractions by performing
-            abstract Interactions while the details of the Protocol Bindings are
-            managed automatically by the WoT Runtime.
+            Here, the <a>WoT Runtime</a> is also a <a>Scripting Runtime</a> system that
+            in addition to managing the WoT-specific aspects also interprets and executes the application scripts.
         </p>
         <figure id="architecture-implementation">
-            <img src="images/wot-thing-scripting.png"
-                style="width: 640px;" />
-            <figcaption>Implementation of a Servient
-                Using the Optional WoT Scripting API.
-            </figcaption>
+            <img src="images/wot-thing-scripting.png" style="width: 80%;" />
+            <figcaption>Implementation of a Servient Using the Optional WoT Scripting API</figcaption>
         </figure>
-        <p> The role and functionality of each additional module shown
+        <p>
+            <a>Servients</a> with <a>WoT Scripting API</a> support usually run on powerful devices, edge nodes, or in the cloud.
+            There are, however, also <a>Scripting Runtimes</a> for resource-constrained systems.
+            Furthermore, the WoT Architecture does not limit <a>WoT Scripting API</a> definitions to JavaScript.
+        </p>
+        <p>
+            The role and functionality of each internal module shown
             in <a href="#architecture-implementation"></a> is given in
             the following sections.
         </p>
-
         <section>
-            <h2>Applications</h2>
-            <p> <a>Applications</a> running on a <a>Servient</a> provide
-                implementations of interactions (for example, what the
-                Servient actually does when a certain action is invoked)
-                and optionally other behaviors, such as control loops or
-                support for local user interfaces. 
-                The <a>WoT Scripting API</a> is one way to
-                implement an application using an application script,
-                and is described in more detail below. Application
-                scripts run inside a <a>Scripting Runtime</a> nested
-                inside the <a>WoT Runtime</a> but have access to objects and
-                interfaces defined by <a>WoT Runtime</a>.
+            <h2>Behavior Implementation</h2>
```

Comment:
Suggest to rename the title to "Implementation of the behavior of a thing"

Reply:
It is not limited to Things, it is behavior implementation in general. It once was "application", but was changed to "behavior" and hence "behavior implementation" in this context. I was never so happy about this wording.